src/Illuminate/Cache/RedisStore.php

```diff
@@ -272,6 +272,8 @@ protected function serialize($value)
      */
     protected function unserialize($value)
     {
-        return is_numeric($value) ? $value : unserialize($value);
+        $unserialized = @unserialize($value);
```

Comment:
Useless unserialization. You could just do:
`return is_numeric($value) || is_string($value) ? $value : @unserialize($value);`